common.sh

```diff
@@ -4,11 +4,11 @@
 OPENSHIFT_RHCOS_MAJOR_REL=""
 OPENSHIFT_RHCOS_MINOR_REL=""
 
-if [[ -z $OPENSHIFT_RHCOS_MAJOR_REL || (! $OPENSHIFT_RHCOS_MAJOR_REL =~ (4.1|4.2|4.3|4.4|latest)) ]]; then
-    OPENSHIFT_RHCOS_MAJOR_REL="4.3"
+if [[ -z $OPENSHIFT_RHCOS_MAJOR_REL || (! $OPENSHIFT_RHCOS_MAJOR_REL =~ (4.1|4.2|4.3|4.4|4.5|latest)) ]]; then
+    OPENSHIFT_RHCOS_MAJOR_REL="4.4"
```

Comment:
I think we should leave this as 4.3 for now, because 4.4 requires 3 masters to successfully deploy a cluster.  My concern is that people are using this to deploy 1-master clusters without explicitly specifying 4.3 or earlier.  So in those cases, if we set the default to 4.4, suddenly their deployments would fail and they might not understand why.